tests/integrations/pd/test_rpc_client.rs

```diff
@@ -366,8 +366,8 @@ fn restart_leader(mgr: SecurityManager) {
     server.stop();
     server.start(&mgr, eps);
 
-    // The GLOBAL_RECONNECT_INTERVAL is 0.1s so sleeps 0.2s here.
-    thread::sleep(Duration::from_millis(200));
+    // The default retry interval is 1s so sleeps 2s here.
+    thread::sleep(Duration::from_secs(2));
```

Comment:
we can still use 100ms for test to reduce test time.

Reply:
Sure